src/pages/SignUpResearcher.js

```diff
@@ -31,17 +32,21 @@ function SignUpResearcher() {
   const classes = useStyles();
   const [sent, setSent] = React.useState(false);
 
-  const validate = (values) => {
+  const validate = values => {
     const errors = required(
       ["organization", "researchArea", "shortIntroduction"],
       values
-    )
+    );
 
     return errors;
   };
 
-  const handleSubmit = (values) => {
+  const handleSubmit = values => {
     //TODO: authenticate to allow login
+
+    // Db.addResearchData(useContext(UserContext)?.uid, ...); TODO
+    // navigate to next page
+
```

Comment:
@CedricLSM You can add pass in the additional data for researcher in here and similarly in the `handleSubmit` function of SignUpVolunteer